### PR TITLE
DOCS-2786: Improve viam-micro-server setup guide

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/installation/micro-rdk-dev.md [hunk 1]
```diff
@@ -1,333 +0,0 @@
----
```
Comment (npentrel) [inline docs/installation/micro-rdk-dev.md:1]: This is mostly a rename, not sure why it's not picking up on that.

File: docs/architecture/viam-micro-server.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+{{% relatedcard link="/components/generic/" %}}
+{{% relatedcard link="/services/data/capture/" %}}
+{{< /cards >}}
+
+Click on each supported resource to see available models, API methods, and configuration info.
+
+## Next steps
+
+To use `viam-micro-server`, follow the installation guide.
+If you want to extend the functionality of `viam-micro-server` or customize it, see the development setup guide.
```
Comment (sguequierre) [inline docs/architecture/viam-micro-server.md:41]:
```suggestion
If you want to access camera functionality, extend the functionality of `viam-micro-server, or customize it see the development setup guide.
```
Might be worth clarifying that here

File: docs/components/camera/esp32-camera.md [hunk 3]
```diff
@@ -29,9 +29,9 @@ For example, the `OV2640` with a ribbon cable connected to an ESP32:
 {{< imgproc src="/components/camera/esp32-camera-2640.png" alt="Fake Camera on the ESP32" resize="300x" >}}
 
 {{< alert title="Software requirements" color="note" >}}
-To use this model, you must follow the [Micro-RDK Development Setup](/installation/micro-rdk-dev/), which enables you to install and activate the ESP-IDF.
-At the step [Generate a new project from a template](/installation/micro-rdk-dev/#generate-a-new-project-from-a-template) where you create a new project with `cargo generate`, select the option to include camera module traits when prompted.
-Finish the [Micro-RDK Development Setup](/installation/micro-rdk-dev/) and return to this guide.
+To use this model, you must follow the [Micro-RDK Development Setup](/installation/viam-micro-server-dev/), which enables you to install and activate the ESP-IDF.
+At the step [Generate a new project from a template](/installation/viam-micro-server-dev/#install-viam-micro-server) where you create a new project with `cargo generate`, select the option to include camera module traits when prompted.
+Finish the [Micro-RDK Development Setup](/installation/viam-micro-server-dev/) and return to this guide.
```
Comment (sguequierre) [inline docs/components/camera/esp32-camera.md:34]:
```suggestion
Finish the [`viam-micro-server` Development Setup](/installation/viam-micro-server-dev/) and return to this guide.
```

File: docs/components/camera/esp32-camera.md [hunk 4]
```diff
@@ -29,9 +29,9 @@ For example, the `OV2640` with a ribbon cable connected to an ESP32:
 {{< imgproc src="/components/camera/esp32-camera-2640.png" alt="Fake Camera on the ESP32" resize="300x" >}}
 
 {{< alert title="Software requirements" color="note" >}}
-To use this model, you must follow the [Micro-RDK Development Setup](/installation/micro-rdk-dev/), which enables you to install and activate the ESP-IDF.
-At the step [Generate a new project from a template](/installation/micro-rdk-dev/#generate-a-new-project-from-a-template) where you create a new project with `cargo generate`, select the option to include camera module traits when prompted.
-Finish the [Micro-RDK Development Setup](/installation/micro-rdk-dev/) and return to this guide.
+To use this model, you must follow the [Micro-RDK Development Setup](/installation/viam-micro-server-dev/), which enables you to install and activate the ESP-IDF.
```
Comment (sguequierre) [inline docs/components/camera/esp32-camera.md:32]:
```suggestion
To use this model, you must follow the [`viam-micro-server` Development Setup](/installation/viam-micro-server-dev/), which enables you to install and activate the ESP-IDF.
```

File: docs/installation/viam-micro-server-dev.md [hunk 5]
```diff
@@ -0,0 +1,323 @@
...
+   brew install viamrobotics/brews/canon
+   ```
+
+{{% /tab %}}
+{{% /tabs %}}
+
+{{% /tab %}}
+{{% tab name="Manual" %}}
+
+To set up the Docker development environment for ESP manually, complete the following instructions:
```
Comment (sguequierre) [inline docs/installation/viam-micro-server-dev.md:132]:
```suggestion
To set up the Docker development environment for ESP32 manually, complete the following instructions:
```
Just bc we don't tend to refer to it as ESP (espressif)

File: docs/installation/viam-micro-server-dev.md [hunk 6]
```diff
@@ -0,0 +1,323 @@
...
+1. Create a new machine
+
+   Navigate to [the Viam app](https://app.viam.com) and [add a new machine](/cloud/machines/#add-a-new-machine) in your desired location.
+
+   Click on the name of the machine to go to the machine's page.
+
+1. [Start Docker](https://docs.docker.com/config/daemon/start/) on your development machine.
+   If you haven't already, complete Docker's [Linux post installation steps](https://docs.docker.com/engine/install/linux-postinstall/) to set up Docker to run whenever your system boots up.
+
+1. Generate a new project from [this template](https://github.com/viamrobotics/micro-rdk/tree/main/templates/project) to create a new micro-RDK project to upload to your ESP32 by running:
```
Comment (sguequierre) [inline docs/installation/viam-micro-server-dev.md:229]:
```suggestion
1. Generate a new project from [this template](https://github.com/viamrobotics/micro-rdk/tree/main/templates/project) to create a new `viam-micro-server` project to upload to your ESP32 by running:
```

**Review by sguequierre — CHANGES_REQUESTED** [review]
Fixing some moments where micro-RDK is still used

**Comment by npentrel** [comment]
Since feedback was fairly minimal, going to merge, that should make the other viam-micro-server ticket easier.

**Pull request merged**